src/test/java/se/sundsvall/messaging/api/model/request/SmsRequestConstraintValidationTests.java

```diff
@@ -73,4 +75,18 @@ void shouldFailWithBlankMessage() {
 		assertThat(validRequest.withMessage(""))
 			.hasSingleConstraintViolation("message", "must not be blank");
 	}
+
+	@ParameterizedTest
+	@ValueSource(strings = {"ab", "1abc", "A_123456", "Abcdefghijklmnop"})
+	void shouldFailWithInvalidSender(final String sender) {
+		assertThat(validRequest.withSender(sender))
+			.hasSingleConstraintViolation("sender", "sender must be between 3-11 characters and start with a non-numeric character");
+	}
+
+	@ParameterizedTest
+	@ValueSource(strings = {"abc", "abc12", "Min Bank"})
```

Comment:
Add or change existing value `Min Bank` to verify maximum threshold for a valid value (11 characters)